test/kata6.joinNames.test.js

```diff
@@ -1,5 +1,17 @@
 const { joinNames } = require("../src");
 
 describe("joinNames", () => {
-  test("returns string of names, seperated by commas and an ampersand", () => {});
+  it("returns string of names, seperated by commas and an ampersand", () => {
+    const bart = {
+      nom: "Bart",
+    };
+    const lisa = {
+      nom: "Lisa",
+    };
+    const maggie = {
+      nom: "Maggie",
+    };
+
```

Comment:
This is a bit over verbose for what this is, its literally just an array of names to be passed in so you could declare some arrays like
```
  const nameSet1 = ['Bart', 'Lisa', 'Maggie'];
  const nameSet2 = ['Fry', 'Leela', 'Zoidberg', 'Amy', 'Scruffy', 'Hermes', 'Farnsworth']; 
```
and pass them into the joinNames function:
```
  expect(joinNames(nameSet1)).toEqual("Bart, Lisa & Maggie");
  expect(joinNames(nameSet2)).toEqual("Fry, Leela, Zoidberg, Amy, Scruffy, Hermes & Farnsworth");
```
This would mean small change to the function under test too, but I think it makes sense as creating an object to only hold a single value isnt really necessary